python/pyspark/sql/streaming/stateful_processor.py

```diff
@@ -102,7 +102,31 @@ def getValueState(self, state_name: str, schema: Union[StructType, str]) -> Valu
             The schema of the state variable. The value can be either a
             :class:`pyspark.sql.types.DataType` object or a DDL-formatted type string.
         """
-        self.stateful_processor_api_client.get_value_state(state_name, schema)
+        self.stateful_processor_api_client.get_value_state(state_name, schema, None)
+        return ValueState(ValueStateClient(self.stateful_processor_api_client), state_name, schema)
+
+    def getValueStateWithTTL(
```

Comment:
Should we keep the same API (`getValueState`) for python and set `ttlDurationMs` as optional? Scala uses method overloading with same API name, I remember we have a similar implementation for [Avro](https://github.com/apache/spark/blob/9b9a7a7478d17222ab92a346aa5d4de31d71349f/connector/avro/src/main/scala/org/apache/spark/sql/avro/functions.scala#L42-L66) `from_avro`/`to_avro` and we use the same API for [python](https://github.com/apache/spark/blob/9b9a7a7478d17222ab92a346aa5d4de31d71349f/python/pyspark/sql/avro/functions.py#L35). cc @HyukjinKwon

Reply:
Yeah, let's keep it same.